docs/game.js

```diff
@@ -0,0 +1,84 @@
+var Frame = require('../docs/frame');
+
+class Game {
+
+  constructor(){
+  this.frames = [];
+  this.rollsTotals = [];
+  this.bonuses = [0,0,0,0,0,0,0,0,0,0,0,0];
+  this.totalsAfterBonuses = [];
+  }
+
+  getFrames() {
+    return this.frames;
+  }
+
+  getRollsTotals() {
+    return this.rollsTotals;
+  }
+
+  getBonuses() {
+    return this.bonuses;
+  }
+
+  gettotalAfterBonuses() {
+    return this.totalAfterBonuses;
+  }
+
+  // creates a frame with the rolls enteres and pushes into the frames array.
+  enterFrame(roll1, roll2){
+    var frame = new Frame(roll1, roll2);
+    this.frames.push(frame.getRolls());
+    this.rollsTotals.push(frame.calcRollsTotal());
+    this.calcFrameBonuses();
+    this.calcTotalsAfterBonuses();
+  }
+
+  calcFrameBonuses(){
+    for (var i=0; i < this.frames.length; i++){
+      if (i % 2 === 0 && this.frames[i] === 10)
+        this.addStrikeBonus(i);
+      else if (this.rollsTotals[i] === 10)
+        this.addSpareBonus(i);
+    }
+  }
+
```

Comment:
I don't quite understand the %2, maybe a comment or a small private method extraction could make it easier to read? making bowling logic easy to read is definitely a nightmare!!